src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -129,14 +129,16 @@ internal static string ReturnCommonPathRootFromTargetSpecifiersIfOneExists(IEnum
 
         public override int Run(AnalyzeOptions analyzeOptions)
         {
-            if (!Environment.GetCommandLineArgs().Any(arg => arg.Equals("--sarif-output-version")))
+            if (!Environment.GetCommandLineArgs().
+                Any(arg => arg.Equals("--sarif-output-version", StringComparison.OrdinalIgnoreCase) ||
+                arg.Equals("-v", StringComparison.OrdinalIgnoreCase)))
             {
-                analyzeOptions.SarifOutputVersion = Sarif.SarifVersion.Current;
```

Comment:
>SarifOutputVersion [](http://example.com/codeflow?start=31&length=18)

put a check here for use of Sarif.SarifVersion.OneZeroZero and just throw an ArgumentException or InvalidOperationException in that case.

'BinSkim no longer supports emitting SARIF 1.0 (an obsolete format). Pass 'Current' on the command-line or omit this argument entirely.'

Reply:
I thought about this simple throw as well, I think my way is better because simple throw will still have the wrong info in the tool help.

Let me change to simple throw.

Reply:
Changed.

Reply:
I see your point! We should be thinking about where this area is going. I think the right thing to do moving forward is to actually eliminate this argument entirely, i.e., we don't want the driver SDK itself to provide for SARIF v1 support by default. And so that argues for taking this approach: the help will be broken, but a future update of the driver SDK could drop this argument entirely.